ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -439,7 +454,22 @@ public int FindTargetForTouch(int reactTag, double touchX, double touchY)
             }
 
             var uiElement = view.As<UIElement>();
+#if WINDOWS_UWP
             var target = VisualTreeHelper.FindElementsInHostCoordinates(new Point(touchX, touchY), uiElement)
+#else
+            var sources = new List<DependencyObject>();
+            VisualTreeHelper.HitTest(
+                uiElement,
+                null,
+                new HitTestResultCallback(
+                    (HitTestResult hit) =>
```

Comment:
You should just use type inference here, remove the `new HitTestResultCallback` constructor wrapper and also remove the (**HitTestResult** hit) parameter type from the lambda parameters.